pkgs/by-name/ur/uradvd/package.nix

```diff
@@ -23,6 +31,12 @@ stdenv.mkDerivation {
     runHook postInstall
   '';
 
+  passthru.tests.version = testers.testVersion {
+    package = uradvd;
+    command = "uradvd --version";
+    version = "uradvd ${finalAttrs.version}";
+  };
```

Comment:
Have you tried using `versionCheckHook` as proposed in https://github.com/NixOS/nixpkgs/pull/443901#issuecomment-3312157531?
If so, what was the issue?

Reply:
Or is the versionCheckHook considered superior to the version passthru test, nevertheless?
If so, why?

Reply:
found out why: https://aux-docs.pyrox.pages.gay/Nixpkgs/Standard-Environment/passthru.chapter/

will resolve this immediately

Reply:
`versionCheckHook` runs at build time, whereas `passthru.tests.version` need to be run separately.
Yes, the test works with a "contains" strategy, but this is good enough.